processes/addFulfillManifest.py

```diff
@@ -33,51 +25,42 @@ def __init__(self, *args):
         self.s3Bucket = os.environ['FILE_BUCKET']
+        self.host = os.environ['DRB_API_HOST']
+        self.port = os.environ['DRB_API_PORT']
+        self.objKey = 'manifests/UofM/'
         self.createS3Client()
 
     def runProcess(self):
         if self.process == 'daily':
             startTimeStamp = datetime.now(timezone.utc) - timedelta(days=1)
-            self.updateManifest(startTimeStamp)
+            self.getManifests(startTimeStamp)
         elif self.process == 'complete':
-            self.updateManifest()
+            self.getManifests()
         elif self.process == 'custom':
             timeStamp = self.ingestPeriod
             startTimeStamp = datetime.strptime(timeStamp, '%Y-%m-%dT%H:%M:%S')
-            self.updateManifest(startTimeStamp)
+            self.getManifests(startTimeStamp)
 
-    def updateManifest(self, startTimeStamp=None):
+    def getManifests(self, startTimeStamp=None):
 
         '''Load batch of LA works from past 24 hours'''
 
-        batches = self.load_batch()
-        for batch in batches:
-            for c in batch['Contents']:
-                if c["LastModified"] > startTimeStamp.replace(tzinfo=timezone.utc):
-                    currKey = c['Key']
-                    metadataObject = self.getObjectFromBucket(currKey, self.s3Bucket)
-                    self.update_batch(metadataObject, self.s3Bucket, currKey)
-
-    def update_batch(self, metadataObject, bucketName, currKey):
-        dbManager = DBManager(
-        user=os.environ.get('POSTGRES_USER', None),
-        pswd=os.environ.get('POSTGRES_PSWD', None),
-        host=os.environ.get('POSTGRES_HOST', None),
-        port=os.environ.get('POSTGRES_PORT', None),
-        db=os.environ.get('POSTGRES_NAME', None)
-    )
+        batches = self.load_batches(self.objKey, self.s3Bucket)
+        filtered_batches = batches.search(f"Contents[?to_string(LastModified) > '\"{startTimeStamp}\"'].Key")
+        for batch in filtered_batches:
+            currKey = batch['Key']
+            metadataObject = self.getObjectFromBucket(currKey, self.s3Bucket)
+            self.update_manifest(metadataObject, self.s3Bucket, currKey)
 
-        dbManager.generateEngine()
-
-        dbManager.createSession()
+    def update_manifest(self, metadataObject, bucketName, currKey):
 
         metadataJSON = json.loads(metadataObject['Body'].read().decode("utf-8"))
         metadataJSONCopy = copy.deepcopy(metadataJSON)
         
-        metadataJSON = self.linkFulfill(metadataJSON, dbManager)
-        metadataJSON = self.readingOrderFulfill(metadataJSON, dbManager)
-        metadataJSON = self.resourceFulfill(metadataJSON, dbManager)
-        metadataJSON = self.tocFulfill(metadataJSON, dbManager)
+        metadataJSON = self.linkFulfill(self, metadataJSON)
+        metadataJSON = self.readingOrderFulfill(self, metadataJSON)
+        metadataJSON = self.resourceFulfill(self, metadataJSON)
+        metadataJSON = self.tocFulfill(self, metadataJSON)
```

Comment:
Don't need to pass `self` as the first argument here

Reply:
You're right. Forgot to change this.